GradientDescent.ipynb

```diff
@@ -42,18 +49,37 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Not implemented.\n"
+     ]
+    },
+    {
+     "ename": "AttributeError",
+     "evalue": "'NotImplementedType' object has no attribute 'reshape'",
+     "output_type": "error",
```

Comment:
Maybe this code block could have the error cleared? Small nit, doesn't need to be fixed